### PR TITLE
Fixing minor lint errors

### PR DESCRIPTION
**Pull request by larryshamalama**
Just some quick pre-commit fixes as a follow-up to #131

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -2,11 +2,6 @@
 # See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 version: 2
 
-build:
```
Comment (ricardoV94) [inline .readthedocs.yml:5]: Are you sure about the RTD one?

**Review by ColtAllen — COMMENTED** [review]
> Just some quick pre-commit fixes as a follow-up to #131

This is my fault 🙁 I had some workstation issues during that PR and couldn't run pre-commit. I'll be buying a new laptop for sure before starting on the PR for https://github.com/pymc-labs/pymc-marketing/issues/127.

I've the same question as @ricardoV94 regarding RTD, otherwise no other comments.

File: .readthedocs.yml (hunk 1 above)
Comment (larryshamalama) [reply .readthedocs.yml:5]: https://github.com/pymc-labs/pymc-marketing/blob/1da568216ad071bc4b7ee034c551e12f73f68d18/.readthedocs.yml#L5-L16

I was getting the following warning that it's duplicated:

image

**Comment by larryshamalama** [comment]
> > Just some quick pre-commit fixes as a follow-up to #131
> 
> This is my fault 🙁 I had some workstation issues during that PR and couldn't run pre-commit. I'll be buying a new laptop for sure before starting on the PR for #127.

Oh, no biggie! I was just working on another issue when I saw some tests failing and it's a super quick fix 🙂

File: .readthedocs.yml (hunk 1 above)
Comment (larryshamalama) [reply .readthedocs.yml:5]: > Are you sure about the RTD one?

In other words, not really...

Comment (ricardoV94) [reply .readthedocs.yml:5]: CC @OriolAbril

Comment (OriolAbril) [reply .readthedocs.yml:5]: Yes, that should only be there one time

**Comment by ricardoV94** [comment]
Thanks Larry

**Pull request merged**